packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -150,6 +150,55 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ** See code in examples/api/lib/widgets/gesture_detector/gesture_detector.1.dart **
 /// {@end-tool}
 ///
+/// ## Troubleshooting
+///
+/// Why isn't my parent [GestureDetector] onTap method called?
+///
+/// Given a parent [GestureDetector] which defines an onTap callback and a child
+/// [GestureDetector] that also defines an onTap callback. When the inner
+/// [GestureDetector] is tapped, both [GestureDetector]s send a [GestureRecognizer]
+/// into the gesture arena because the pointer coordinates are in both
+/// [GestureDetector]s bounds. The child [GestureDetector] wins because it was
+/// the first detector to enter the arena and it got first dips because it is in front.
+/// For more information on gesture desambiguation see:
+/// [Gesture disambiguation](https://docs.flutter.dev/development/ui/advanced/gestures#gesture-disambiguation)
+///
+/// Setting [GestureDetector.behavior] to [HitTestBehavior.opaque]
+/// or [HitTestBehavior.translucent] has no impact on parent-child relationships
+/// like in this case. To block hits on children use an [IgnorePointer] or [AbsorbPointer].
+///
+/// Some callbacks are called during the gesture arena resolution
+/// (for instance onTapDown). If the parent [GestureDetector] defines such callbacks,
+/// they will be called. Some callbacks are conditionnaly called, for instance
+/// [TapGestureRecognizer.onTapDown] triggers once a short timeout has elapsed.
+///
+/// {@tool dartpad}
```

Comment:
All of the breadcrumbs in this sample description make it hard to read. I think typically (for a given section of documentation) the first mention of a `[class]` is a breadcrumb, and any subsequent mentions of `class` are not.

Reply:
(for the ones that are not, probably we should use \`...\`?)